num-codex/codex-feasibility-backend
# #44 -  refactor database

**Still work in progress**

**What's in the PR**

- database design has been changed and java classes have been modified accordingly

**How to test manually**
- there is no migration path from the old database. so you need to start over.
- see the curl examples from the readme and check if this works as intended
